steps/reasons-for-appealing/other-reasons-for-appealing/OtherReasonForAppealing.js

```diff
@@ -19,10 +19,9 @@ class OtherReasonForAppealing extends Question {
 
         return form(
 
-            textField('otherReasonForAppealing').joi(
-                this.content.fields.otherReasonForAppealing.error.invalid,
-                Joi.string().allow('').regex(whitelist)
-            )
+            textField('otherReasonForAppealing')
+                .joi(this.content.fields.otherReasonForAppealing.error.invalid,
+                    Joi.string().regex(whitelist).allow('').required())
```

Comment:
This field is optional please remove the required(). For consistency please keep the formatting the same as other steps.

Reply:
done.